packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/ConfirmationDialog.tsx

```diff
@@ -147,20 +157,37 @@ const ConfirmationDialog = ({
     plan
   );
 
+  const ariaLabelledByError = "error-content-header";
+  const ariaDescribedByError = "error-content-description";
+
+  const ariaLabelledByConfirmation = "confirmation-content-header";
+  const ariaDescribedByConfirmation = "confirmation-content-description";
+
+  let ariaLabelledBy;
+  let ariaDescribedBy;
+
+  ariaLabelledBy = !loading && error ? ariaLabelledByError : ariaLabelledByConfirmation;
+  ariaDescribedBy = !loading && !error ? ariaDescribedByError : ariaDescribedByConfirmation;
```

Comment:
Q: Should we just set these above on L166-167 and change `let` 👉 `const` since the values don't seem to change?